src/Mvc/MvcTests.ruleset

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Rules for MVC Tests" Description="Code analysis rules for MVC Tests" ToolsVersion="15.0">
-  <Rules AnalyzerId="xunit.analyzers" RuleNamespace="xunit.analyzers">
-    <!-- 
-      This code is produced when theory parameters that are unused. Normally, this is a warning which causes our build to fail.
-    -->
-    <Rule Id="xUnit1026" Action="Info" />
```

Comment:
There are some violations of this rule. The rule should be enabled and violations should get fixed as they make tests very confusing.

For example, the "expected" parameter of a test isn't used.

However, this is out of scope for this PR.

Do you want me to open a tracking  issue?

Reply:
> Do you want me to open a tracking issue?

No, this isn't an issue we can actually fix. Some test parameters are used for their types and not their values. Fortunately, there should be very few violations.

Reply:
@dougbu Can you elaborate more?

Here:

https://github.com/dotnet/aspnetcore/blob/c925f99cddac0df90ed0bc4a07ecda6b054a0b02/src/Mvc/Mvc.Formatters.Xml/test/SerializableErrorWrapperProviderTest.cs#L11-L21

`isSerialization` parameter is never used. The test might be wrong if it should use it, or it should be removed.

Reply:
I didn't say nothing could be fixed😁